src/components/send/SendAssetFormToken.js

```diff
@@ -7,7 +7,7 @@ import { useDimensions } from '@rainbow-me/hooks';
 import { supportedNativeCurrencies } from '@rainbow-me/references';
 import { removeLeadingZeros } from '@rainbow-me/utils';
 
-const footerMargin = 30 + getSoftMenuBarHeight() / 2;
+const footerMargin = getSoftMenuBarHeight() / 2;
```

Comment:
this is modifying both OSs, was it on purpose?

Reply:
https://cloud.skylarbarrera.com/Screen-Shot-2021-12-14-09-03-24.80.png